File: code-examples/php/usage-examples/command.php
<?php

require 'vendor/autoload.php';

// Replace the uri string with your MongoDB deployment's connection string
$uri = "mongodb://localhost:27017";

try {
    $client = new MongoDB\Client($uri);

    // Print the storage statistics for the "sample_mflix" database by using the 'dbStats' command
    $command = new MongoDB\Driver\Command(["dbStats" => 1]);
    $result = $client->getManager()->executeCommand("sample_mflix", $command);

    var_dump($result->toArray());
} catch(Exception $e) {
    // If an error occurs it will be captured by the catch block and logged to the console
    error_log($e->getMessage());
}

?>

File: code-examples/scala/usage-examples/command.scala
import org.mongodb.scala._

// Replace the uri string with your MongoDB deployment's connection string
val uri: String = "<connection string uri>"

val client: MongoClient = MongoClient(uri)

try {

  // Get the "sample_mflix" database
  val db: MongoDatabase = client.getDatabase("sample_mflix")

  // Find and print the storage statistics for the "sample_mflix" database using the 'dbStats' command
  val result = db.runCommand(Document("dbStats" -> 1)).headResult()

  println(result)

} catch {

  // Print the stack trace of any errors that occurred
  case e: Throwable => e.printStackTrace

} finally {

  // Close the database connection on completion or error
  client.close()

}
